__tests__/app.test.js

```diff
@@ -218,6 +218,25 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("200: responds with array of objects with required fields", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const { allUsers } = body;
+        expect(allUsers).toHaveLength(4);
+        allUsers.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
```

Comment:
Nice work on this test - great use of toMatchObject and I like that you have remembered to check the length of the array which is really important when we assert inside a forEach like this :)